src/test/mochitest/browser_dbg-preview-module.js

```diff
@@ -0,0 +1,27 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Test hovering in a script that is paused on load
+// and doesn't have functions.
+add_task(async function() {
+  const dbg = await initDebugger("doc-scripts.html");
+  const { selectors: { getSelectedSource }, getState } = dbg;
+
+  navigate(dbg, "doc-on-load.html")
+
+  await waitForPaused(dbg);
```

Comment:
could you add a comment to say that the test page has a debugger statement that will pause the debugger, hence the waitForPaused here ?